resources/views/checkout/partials/summary.blade.php

```diff
@@ -0,0 +1,116 @@
+                            @{{ option.option_label }}: @{{ option.value_label }}
+                        </div>
+                        <div v-for="option in item.customizable_options">
+                            @{{ option.label }}: @{{ option.values[0].label || option.values[0].value }}
+                        </div>
+                        <div v-for="option in config.cart_attributes">
+                            <template v-if="item.product.attribute_values?.[option] && typeof item.product.attribute_values[option] === 'object'">
+                                @{{ window.attributeLabel(option) }}: <span v-html="item.product.attribute_values[option]?.join(', ')"></span>
+                            </template>
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="w-2/12 text-right">@{{ item.quantity }}</div>
+            <div class="w-3/12 text-right">@{{ item.prices.row_total.value | price }}</div>
+        </dt>
+    </x-rapidez::summary.row>
+
+    <x-rapidez::summary.divider />
+
+    <x-rapidez::summary.row>
+        <x-rapidez::summary.title>@lang('Subtotal')</x-rapidez::summary.title>
+        <x-rapidez::summary.value>
+            @{{ cart.prices.subtotal_including_tax.value | price }}
+        </x-rapidez::summary.value>
+    </x-rapidez::summary.row>
+
+    <template v-if="cart.shipping_addresses.length && cart.shipping_addresses[0]?.selected_shipping_method">
+        <x-rapidez::summary.divider />
+        <x-rapidez::summary.row>
+            <x-rapidez::summary.title>@lang('Shipping')</x-rapidez::summary.title>
+            <x-rapidez::summary.value>@{{ cart.shipping_addresses[0]?.selected_shipping_method.amount.value | price }}</x-rapidez::summary.value>
+            <x-rapidez::summary.description>@{{ cart.shipping_addresses[0]?.selected_shipping_method.carrier_title }} - @{{ cart.shipping_addresses[0]?.selected_shipping_method.method_title }}</x-rapidez::summary.description>
+        </x-rapidez::summary.row>
+    </template>
+
+    <x-rapidez::summary.divider />
+
+    <template v-for="discount in cart.prices.discounts">
+        <x-rapidez::summary.row>
+            <x-rapidez::summary.title>@{{ discount.label }}</x-rapidez::summary.title>
+            <x-rapidez::summary.value class="text-green-700 font-semibold">-@{{ discount.amount.value | price }}</x-rapidez::summary.value>
+        </x-rapidez::summary.row>
+    </template>
+
+    <x-rapidez::summary.row>
+        <x-rapidez::summary.title class="font-bold text-base">@lang('Total')</x-rapidez::summary.title>
+        <x-rapidez::summary.value class="font-bold text-base">@{{ cart.prices.grand_total.value | price }}</x-rapidez::summary.value>
+        <x-rapidez::summary.description v-if="cart.prices.applied_taxes?.[0]?.amount?.value" class="text-right pr-0">
+            @lang('Inclusive') @{{ (cart.prices.applied_taxes?.[0]?.amount?.value || 0) | price }} @lang('Tax')
```

Comment:
```suggestion
            @lang('Including') @{{ (cart.prices.applied_taxes?.[0]?.amount?.value || 0) | price }} @lang('Tax')
```

(also change this in the translation file btw)